M2/Macaulay2/e/text-io.hpp

```diff
@@ -4,13 +4,19 @@
 
 #include "buffer.hpp"
 #include "engine-includes.hpp"
+#if USING_MPIR 
+#include <mpir.h>
+#include <mpirxx.h>
+#else
 #include <gmp.h>
+#include <gmpxx.h>
+#endif
```

Comment:
Should this bit be included in `engine-includes.hpp` so everything can include it?

Reply:
Specifically, I'm confused how some files have this chunk and some don't (and I can't tell if they're including it via another file)

Reply:
@mikestillman -- this is a question for you